janis_bioinformatics/tools/gatk4/collectinsertsizemetrics/base.py

```diff
@@ -203,3 +211,38 @@ def outputs(self):
             doc="display hidden  arguments  Default  value: false.  Possible values: {true, false} ",
         ),
     ]
+
+    def tests(self):
+        with open(
+            os.path.join(
+                BioinformaticsTool.test_data_path(),
+                "wgsgermline_data",
+                "NA12878-BRCA1.markduped.metrics.txt",
+            ),
+            "r",
+        ) as f:
+            for i in range(5):
```

Comment:
Just to be clearer, can we add some comment about the significance of the first 5 lines of this file?

Reply:
I've added comments to explain why I did this. It's mainly because the headers in the first 5 lines change with every run. But still 5 is very arbitrary. Any other number bigger than 5 would serve the same goal. Also, I could have avoided reading another file and only check if the output contains a certain line but I just decided to try something different here.